### DISCUSSION TITLE
cannot verify my academic affiliation with my student card (japanese) and enrollment certificate (english)

### DISCUSSION THREAD

**Discussion by zfhrp6** [community/community#120453] [GitHub Education] [answered]
### Hi everyone,

I can't apply. please help.

## what i did

applying

* with student card (in Japanese)
* with enrollment certificate (in English and in Japanese)

### with my student card

the card has followings:

* the name of the school (in Japanese)
* my name (first and last) (in Japanese)
* my student number
* expiration of the card (in Japanese).

For this time, my name in the billing information is in Japanese.

The message says,

* The image you selected does not appear to contain your school name.
* Please ensure that your academic affiliation document contains your last name exactly as it appears in your GitHub billing information.
* Please ensure that your academic affiliation document contains your first name exactly as it appears in your GitHub billing information.

### with enrollment certificate (in English)

The certificate has followings:

* the name of the school
* name (first and last)
* student number
* date of enrollment
* student number
* my first name and my last name.

For this time, my name in the billing information is in English.

The message says,

* Please ensure that your academic affiliation document contains your last name exactly as it appears in your GitHub billing information.

### with enrollment certificates (in English and in Japanese)

For this time, my name in the billing information is in Japanese.

The process needs a little more, and I uploaded a image of my student card.

Message says as same as with my student card.

**Comment by samus-aran** [comment]
Hi @zfhrp6 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [I need to translate my evidence. What can I do?](https://github.com/orgs/community/discussions/111352#translate-my-evidence)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍

**Reply by zfhrp6** [reply, marked as answer]
Thank you, samus-aran.
As it turns out, translation was not necessary in my case.

The request was approved with a photo of the Enrollment Certificate (in English) taken close enough to clearly show the name of the school, name and other necessary information. The 1MB size limit was a bit difficult😅.

The photo was taken with the message below in mind.

> Please ensure that your academic affiliation document contains your last name exactly as it appears in your GitHub billing information.
